src/universal/modules/teamDashboard/containers/Team/Team.js

```diff
@@ -1,6 +1,17 @@
 import React, {PropTypes} from 'react';
+import {connect} from 'react-redux';
+import {cashay} from 'cashay';
 import Team from 'universal/modules/teamDashboard/components/Team/Team';
 import requireAuth from 'universal/decorators/requireAuth/requireAuth';
+import {getAuthQueryString, authedOptions} from 'universal/redux/getAuthedUser';
+
+const mapStateToProps = (state, props) => {
+  const {params: {id: teamId}} = props;
+  return {
+    teamId,
```

Comment:
This has nothing to do with Redux. Instead, we should use `:teamId` in the router. will fix